docs/web/docs/guides/tutorials/first_task.md

```diff
@@ -236,45 +236,54 @@ Complete this task, and you can review it in the same way as before.
-2. We set up the default `conf` file to be `example`, which ensures that Hydra looks into `conf/example.yaml` on any run where this isn't overridden.
-3. We create a `RunScriptConfig`, which is a set of options that we are allowed to set for a task. Here, we only provide a `task_dir`, which you can override on the command line with `python static_test_script.py task_dir=some/other/path`. Mostly, we use this for ensuring that the `data_csv` and other path-related variables are portable.
-4. We register this config with hydra, and then have the `main` function point to this configuration. Note the `hydra_configs` value for `config_path`, which now fully explains the `hydra_configs/conf/example.yaml` path. 
+1. We import the `BLUEPRINT_TYPE_STATIC_HTML` to force an import of the blueprint we intend to use with this run script. This isn't *required* for Mephisto default blueprints, but is important for custom `Blueprint`s to ensure they are loaded into the Mephisto registry.
+2. We set up the default `conf` file to be `example`, using `build_default_task_config`, which returns a `TaskConfig` that we can extend.
+3. We extend the returned `TaskConfig` with `MyTaskConfig`, which allows us to specify custom arguments.
+4. We decorate the main, noting that the correct config is `MyTaskConfig`. Note that the `default_config_file` version of this simply takes care of the above steps inline in the decorator.
 
 With all the above, we're able to just make edits to `example.yaml` or make other configs in the `conf/` directory and route to them directly.
 
 ### 3.2 Invoking Mephisto
 Mephisto itself is actually invoked just a little later:
 ```python
-def main(cfg: DictConfig) -> None:
-    db, cfg = load_db_and_process_config(cfg)
-    operator = Operator(db)
-
-    operator.validate_and_run_config(cfg.mephisto)
+def main(operator: Operator, cfg: DictConfig) -> None:
+    operator.launch_task_run(cfg.mephisto)
     operator.wait_for_runs_then_shutdown(skip_input=True, log_rate=30)
 ```
-Here we use the `load_db_and_process_config` helper to extract specific arguments out of your configuration (and surface warnings about incompatibilities), as well as load the correct `MephistoDB` for the task. We then initialize an `Operator` (the main entry point for running a Mephisto `TaskRun`), and run the given config, then wait for it to run. To ensure we're not frozen, the operator takes in a `log_rate` in seconds to print status messages while the run is underway.
+Here we use the `process_config_and_get_operator` helper to extract specific arguments out of your configuration (and surface warnings about incompatibilities), as well as initialize an `Operator` on the correct `MephistoDB` for the task. Using this we can launch a `TaskRun` the given config, then wait for it to run. To ensure we're not frozen, the operator takes in a `log_rate` in seconds to print status messages while the run is underway.
```

Comment:
We say `process_config_and_get_operator` here, this is outdated? We are using `launch_task_run`